dhis-2/dhis-web-api/src/main/resources/openapi/EnrollmentsExportController.md

```diff
@@ -26,11 +26,15 @@ for how to use it.
 
 ### `*.parameter.EnrollmentRequestParams.enrolledAfter`
 
-Get enrollments with an enrollment date after the given date.
+Get enrollments enrolled after the given date and time.
```

Comment:
I would put all into one paragraph without newlines.

Does the ISO8601 format show up in the final spec? If not we need to describe that here as well as users should now what format we expect.